test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -186,6 +186,26 @@ Describe "Get-ChildItem" -Tags "CI" {
             $app.Target | Should -Not -Be $app.FullName
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Get-ChildItem.FileSystem provider.Wildcard matching behavior is the same between -Path and -Include parameters when using escape characters
```powershell
Expected exactly 'C:\Users\VssAdministrator\AppData\Local\Temp\3733c351-da8a-4c62-adcc-645bb414dd78\a`[b]', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Get-ChildItem.Tests.ps1: line 200
200:             $WithInclude.Name | Should -BeExactly $expectedPath
```